I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 198, in script call
    call _gl_test
  File "renpy/common/00gltest.rpy", line 440, in script
    with None
  File "game/inventory.rpy", line 53, in display_items_overlay
    for i in range(0, len(inventory.items)):
NameError: global name 'inventory' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "D:\Renpy\renpy-7.0.0-sdk\renpy\bootstrap.py", line 306, in bootstrap
    renpy.main.main()
  File "D:\Renpy\renpy-7.0.0-sdk\renpy\main.py", line 513, in main
    run(restart)
  File "D:\Renpy\renpy-7.0.0-sdk\renpy\main.py", line 139, in run
    renpy.execution.run_context(True)
  File "D:\Renpy\renpy-7.0.0-sdk\renpy\execution.py", line 879, in run_context
    context.run()
  File "renpy/common/00start.rpy", line 198, in script call
    call _gl_test
  File "renpy/common/00gltest.rpy", line 440, in script
    with None
  File "D:\Renpy\renpy-7.0.0-sdk\renpy\ast.py", line 1337, in execute
    renpy.exports.with_statement(trans, paired)
  File "D:\Renpy\renpy-7.0.0-sdk\renpy\exports.py", line 1452, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "D:\Renpy\renpy-7.0.0-sdk\renpy\display\core.py", line 2200, in do_with
    self.with_none()
  File "D:\Renpy\renpy-7.0.0-sdk\renpy\display\core.py", line 2224, in with_none
    self.compute_overlay()
  File "D:\Renpy\renpy-7.0.0-sdk\renpy\display\core.py", line 2321, in compute_overlay
    i()
  File "game/inventory.rpy", line 53, in display_items_overlay
    for i in range(0, len(inventory.items)):
NameError: global name 'inventory' is not defined

Windows-8-6.2.9200
Ren'Py 7.0.0.196
Escape Roomba 0.0
Thu Sep 13 13:28:48 2018
